custom_components/versatile_thermostat/translations/en.json

```diff
@@ -302,6 +305,7 @@
           "heater_entity2_id": "Optional 2nd Heater entity id. Leave empty if not used",
           "heater_entity3_id": "Optional 3rd Heater entity id. Leave empty if not used",
           "heater_entity4_id": "Optional 4th Heater entity id. Leave empty if not used",
+          "heater_keep_alive": "Optional heater switch state refresh interval. Disabled if empty or zero",
```

Comment:
add "Use it only if your switch need it." at the end of the description. Idem for step/option description.

Reply:
I agree that it makes sense to advise the user that this feature should only be used when required, 👍 as it has “performance implications” of sorts. But I also noticed that the whole description was getting long:

```python
"heater_keep_alive": "Optional heater switch state refresh interval. Disabled if empty or zero. Use it only if your switch needs it."
```

 The preceding descriptions, for example, have a more succinct _“Leave empty if not used”_ instruction:
```python
"heater_entity2_id": "Optional 2nd Heater entity id. Leave empty if not used",
"heater_entity3_id": "Optional 3rd Heater entity id. Leave empty if not used",
"heater_entity4_id": "Optional 4th Heater entity id. Leave empty if not used",
```

I had added _"Disabled if empty or zero”_ because my schema definition meant that the form was pre-filled with `0`, so the user needed to be advised of the meaning of `0`:

```python
vol.Optional(CONF_HEATER_KEEP_ALIVE, default=0): cv.positive_int,
```

But from the end user’s point of view, leaving the form empty is more straightforward than entering zero. So I've changed the schema for the default to be an empty field:

```python
vol.Optional(CONF_HEATER_KEEP_ALIVE): cv.positive_int,
```

I have tested to confirm than an empty field doe not cause any validation issues with the `positive_int` constraint, and the Python code replaces `None` with `0` to satisfy the `int` type hint:

```python
 keep_alive_sec=config_entry.get(CONF_HEATER_KEEP_ALIVE, 0)
```

So, there’s no longer a need to mention “zero” to the user. My new proposal is to say _“Leave empty if not required”:_

```python
"heater_keep_alive": "Optional heater switch state refresh interval. Leave empty if not required.",
```

I think this also removes the need to say _”Use it only if your switch needs it.”_ The preceding `heater_entity` descriptions read _”Leave empty **if not used**”_ rather than _”**if not required**”_ and I think that’s appropriate: If you are not _”using”_ an extra switch entity then you leave the field empty _”if not used”,_ whereas if your switch does not _”require”_ keep-alive messages, then you leave the field empty _”if not required”._ Furthermore, while _”Disabled if empty or zero”_ was just a piece of information, _“Leave empty if not required”_ is a more forceful instruction of what the user should do.

Would you agree? If you agree, please let me have amended French translations as well. I will wait for your feedback before updating the screenshots.

Reply:
Great.